build/initialize-pipeline.ps1

```diff
@@ -4,9 +4,11 @@ $bypassPackaging = $true
 $includeSuffix = $true
 Write-Host "SourceBranch: $sourceBranch, Build reason: $buildReason"
 
-if($sourceBranch.endsWith('master') -and ($buildReason -ne "PullRequest"))
+if(($sourceBranch.endsWith('master') -or $sourceBranch.endsWith('dev')) -and ($buildReason -ne "PullRequest"))
 {
-  $includeSuffix = $false
+  if($sourceBranch.endsWith('master')) {
```

Comment:
Why do we include the suffix only for master? Is the suffix runtime version?

Reply:
suffix is whether to include "-prerelease" string or not.